### PR TITLE
bghostview, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
BGhostview

File: haiku-apps/bghostview/bghostview-20190401.recipe [hunk 1]
```diff
@@ -0,0 +1,55 @@
+SUMMARY="A port of KGhostview for X11/KDE"
+DESCRIPTION="This is techically a port of KGhostview for X11/KDE. However \
```
Comment (korli) [inline haiku-apps/bghostview/bghostview-20190401.recipe]: technically

File: haiku-apps/bghostview/bghostview-20190401.recipe [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgs$secondaryArchSuffix >= 9
+	lib:libprefs$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgs$secondaryArchSuffix >= 9
+	lib:libprefs$secondaryArchSuffix
```
Comment (korli) [inline haiku-apps/bghostview/bghostview-20190401.recipe]: Not devel:libprefs$secondaryArchSuffix?

**Comment by pulkomandy** [comment]
> Not devel:libprefs$secondaryArchSuffix?

libprefs package needs to be fixed to provide that first, I think?

File: haiku-apps/bghostview/bghostview-20190401.recipe (hunk 1 above)
Comment (Begasus) [reply haiku-apps/bghostview/bghostview-20190401.recipe]: copy/paste (will fix) :)

**Comment by Begasus** [comment]
Created a PR for the _devel package for libprefs at https://github.com/haikuports/haikuports/pull/3736

**Comment by Begasus** [comment]
This should be ok, if you want the icons you also need xpmtranslator (https://github.com/haikuports/haikuports/pull/3744), this can't be merged yet due to the license but one can build it

**Pull request merged**